input/WIxSC_0.yml

```diff
@@ -0,0 +1,23 @@
+mapper_class: MapperWIxSC
+mask_name: mask_2MPZWISC
+path_rerun: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/xcell_runs'
+coords: G
+z_edges: [0.10, 0.15]
+bin_name: 'z0p10_0p15'
+nside_nl_threshold: 8129
+lmin_nl_from_data: 8192
+nl_analytic: True
+data_catalog: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/WIxSC.fits'
+spec_sample: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/zSpec-comp-WIxSC.csv'
+n_jk_dir: 100
+apply_galactic_correction: True
+mask: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/WISExSCOSmask.fits.gz'
+star_map: '/mnt/extraspace/damonge/Datasets/2MPZ_WIxSC/allwise_total_galactic.fits'
+use_halo_model: True
```

Comment:
same comments as 2MPZ about `hod_params` and `use_halo_model`

Reply:
Applies to all WIxSC

Reply:
same answers